### PR TITLE
Monica/remove route summary

### PR DESCRIPTION
**Pull request by myo3**

### CODE REVIEW

File: TCAT/Model/Route.swift [hunk 1]
```diff
@@ -135,86 +128,11 @@ class Route: NSObject, JSONDecodable {
...
-                    }
-                }
-            }
-        }
-    }
-    
     func isWalkingRoute() -> Bool {
-        for routeSummaryObj in routeSummary {
-            if routeSummaryObj.nextDirection != .walk {
+        for direction in directions {
```
Comment (mattbarker016) [inline TCAT/Model/Route.swift]: `let isWalkingRoute = route.directions.reduce(true) { $0 && $1.type == .walk }`
So Swift. So functional.

File: TCAT/Model/Route.swift [hunk 2]
```diff
@@ -224,38 +142,20 @@ class Route: NSObject, JSONDecodable {
...
-                if lastRouteSummaryObject.name == "End" && routeSummary.count > 2 && routeSummary[secondToLastStop].name.lowercased() == place.name.lowercased() {
-                    routeSummary.removeLast()
-                    routeSummary.last?.removeNextDirection() // remove 2nd-to-last stop next direction (so it behaves like ending stop)
-                } else if(lastRouteSummaryObject.name == "End") {
-                    lastRouteSummaryObject.updateName(from: place)
-                }
-            }
-        }
+        let numberOfMetersInMile = 1609.34
+        let distanceInMeters = fromLocation.distance(from: endLocation)
```
Comment (mattbarker016) [inline TCAT/Model/Route.swift:151]: I'd like to centralize conversions if possible, hmm

**Review by mattbarker016 — COMMENTED** [review]
Nothing technically *has* to be changed 😛

File: TCAT/Model/Route.swift (hunk 1 above)
Comment (myo3) [reply TCAT/Model/Route.swift]: tru tru

File: TCAT/Model/Route.swift (hunk 2 above)
Comment (myo3) [reply TCAT/Model/Route.swift:151]: yeah maybe we can think of a class later to put that in or make it an extension of double?

**Pull request merged**